src/NServiceBus.Core/Transports/Msmq/MsmqDequeueStrategy.cs

```diff
@@ -18,6 +18,8 @@ namespace NServiceBus.Transports.Msmq
     /// </summary>
     public class MsmqDequeueStrategy : IDequeueMessages, IDisposable
     {
+        static ILog Logger = LogManager.GetLogger<MsmqDequeueStrategy>();
```

Comment:
apparently, the code cleanup moves it to the top...wasn't aware of that but let's keep it there then.